functions/.znap.clone.task

```diff
@@ -10,7 +10,7 @@ private repo=~znap/$url:t:r
 [[ -d $repo ]] &&
     return 1
 
-private -a git_config=()
```

Comment:
I prefer the form with `=()`, since it makes it easier to spot the arrays in the code.